src/main/java/com/rybka/service/BaseApiConnectorService.java

```diff
@@ -0,0 +1,7 @@
+package com.rybka.service;
+
+import com.rybka.model.ExchangeResponse;
+
+public interface BaseApiConnectorService {
```

Comment:
Rename to BaseCurrencyExchangeConnector.

Reply:
fixed